Assets/MixedRealityToolkit.Tools/ExtensionServiceCreator/ExtensionServiceCreator.cs

```diff
@@ -246,39 +370,44 @@ public async void LoadStoredState()
             }
         }
 
+        /// <summary>
+        /// Validate template assets
+        /// </summary>
+        /// <remarks>
+        /// Adds to items to errors log field if not valid
+        /// </remarks>
+        /// <returns>true if errors encountered, false otherwise</returns>
         public bool ValidateAssets(List<string> errors)
         {
-            errors.Clear();
```

Comment:
Do we not still want to clear, in case a non-empty list is passed in?

Reply:
No, we don't want to clear. The function will only add to whatever list is given as input. This is more flexible for the consumer. if they want to clear, they can do it themselves since they control the list before calling the function. But if the consumer doesn't want to clear and the function automatically did, then you need to AddRange the results from this function and effectively duplicating work/copying